syntax/encoding/yaml.go

```diff
@@ -59,7 +60,16 @@ func newPositionIndex(yaml []byte) positionIndex {
 	offset, lines := 0, []linePosition(nil)
 	for {
 		line, rest, found := bytes.Cut(yaml, []byte{'\n'})
-		lines = append(lines, linePosition{offset: offset, line: line})
+
+		ascii := true
+		for _, b := range line {
+			if b&0x80 != 0 {
```

Comment:
Is there a reference for this incantation that we could add as a comment?

Reply:
Done